lib/simple_form/action_view_extensions/builder.rb

```diff
@@ -30,8 +30,6 @@ def simple_fields_for(*args, &block)
   end
 end
 
-module ActionView::Helpers
-  class FormBuilder
-    include SimpleForm::ActionViewExtensions::Builder
-  end
+ActiveSupport.on_load(:action_view) do
```

Comment:
This change seems not needed. On load hooks are only for constants that use it. The problem is to load action view base. Loading action view helpers and form builder should not cause any problem.